src/models/facilityStrengthMachine.ts

```diff
@@ -24,6 +24,8 @@ export interface FacilityStrengthMachineData {
 
 export interface FacilityStrengthMachineInitializerTokenResponse extends AuthenticatedResponse {
   initializerToken: string
+  url: string
+  expiresAt?: string
```

Comment:
Make 2 different interfaces with explicit declaration for `expiresAt`. (One can inherit from the other).